client/docs/cli.md

```diff
@@ -0,0 +1,193 @@
+fraudulent                     quickstart                          label
+fraud_training                 quickstart                          training set
+transactions                   kaggle                              source
+average_user_transaction       quickstart                          source
+```
+
+Listed below the user are all the resources registered to that user.
+
+### Example: Getting a Resource
+
+The following command shows how to retrieve information on a specific resource, a **feature** named **"avg\_transactions"**.
+
+```shell
+featureform get feature avg_transactions --host $FEATUREFORM_HOST --cert $FEATUREFORM_CERT
+
+NAME:  avg_transactions
+STATUS:  NO_STATUS
+VARIANTS:
+quickstart           default
+v1
+v2
+prodThis would be the output:
+```
+
+### Example: Getting a Resource Variant
+
+The command below retrieves information on the specific variant of the **feature** "**avg\_transactions", "quickstart"**
+
+```shell
+featureform get feature avg_transactions quickstart --host $FEATUREFORM_HOST --cert $FEATUREFORM_CERT
+
+NAME:                avg_transactions
+VARIANT:             quickstart     
+TYPE:                float32
+ENTITY:              user
+OWNER:               featureformer
+DESCRIPTION:
+PROVIDER:            redis-quickstart
+STATUS:              READY
+
+SOURCE:
+NAME                           VARIANT
+average_user_transaction       quickstart
+
+TRAINING SETS:
+NAME                           VARIANT
+fraud_training                 quickstart
+```
+
+Listed below are the metadata on that variant, as well as its source and the training sets it belongs to.
+
+## List
+
+The **LIST** command displays the name, variant and status of all the resources of that resource type.
+
+```shell
+featureform list RESOURCE_TYPE --host $FEATUREFORM_HOST –cert $FEATUREFORM_CERT
+```
+
+**RESOURCE\_TYPE** (required) can be:
+
+* **features -** machine learning features
+* **labels -** machine learning labels
+* **training-sets -** set of features and one label for training ML models
+* **users -** registered users in your instance
+* **entities -** identifier for a source of features or labels (akin to a primary key)
+* **models -** registered machine learning models which training sets and features are fed to
+* **providers -** registered 3rd party providers which store your data
+* **sources -** files, tables, or transformations that features, labels and training sets source from
+
+NOTE: The **--cert $FEATUREFORM_CERT** is only required for self-signed certs
+
+### Example: Getting the list of users
+
+``` shell
+featureform list users --host $FEATUREFORM_HOST --cert $FEATUREFORM_CERT
+featureform list users --insecure --host $FEATUREFORM_HOST
+```
+
+The commands are both valid ways to retrieve a list of users. The first is when the user uses a self-signed cert.
+
+The following uses the local flag to access resources created and stored in localmode:
```

Comment:
the tiniest nit. "localmode" is missing a space to match "local mode" on line 182. or vice versa